### PR TITLE
refactor: Format deploy chain message

### PR DESCRIPTION
**Pull request by howjmay**
The generated message will be 

```
DKG successful
Address: tst1qpzfcen3pxqr4ysweyg6lpnh4n74f4445swfdd87qke6szprz82puw6ngfe
Committee:
* size=1
* quorum=1
* members: 0x340d3493dfa78e39d3099bb4febe7d4be74e8f0ef354acf64a89f130cf772eb9 (me)

Creating new chain
* Owner address:    0xe36091af1cd2873c1705b92188a2f59b551fd02e12d8d18c3a5b97d4501fe844
* State controller: 0x449c667109803a920ec911af8677acfd54d6b5a41c96b4fe05b3a8082311d41e, N = 1, T = 0
2023-04-12T17:30:29-06:00       INFO    nc      Posted blockID 0x5f4d9e53cff641ba66db392f572be9c07570bb7497a049cf2cdb552557f9b5da
2023-04-12T17:30:29-06:00       INFO    nc      Posted transaction id 0x97183c96add27df23f4cc3838ca8da99083443dcccc495828527a8928a2ba941
Chain has been created successfully on the Tangle.
* ChainID: tst1pzwy7ksq8xatl06g5u7lkn8ujtq9plw8ftyqh6q0nes9nw27artm7n0fq3q
* State address: tst1qpzfcen3pxqr4ysweyg6lpnh4n74f4445swfdd87qke6szprz82puw6ngfe, N = 1, T = 0
Make sure to activate the chain on all committee nodes
Chain: tst1pzwy7ksq8xatl06g5u7lkn8ujtq9plw8ftyqh6q0nes9nw27artm7n0fq3q (mychain)
Activated
```

### CODE REVIEW

File: packages/apilib/deploychain.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func DeployChain(par CreateChainParams, stateControllerAddr, govControllerAddr i
 	originatorAddr := par.OriginatorKeyPair.GetPublicKey().AsEd25519Address()
 
 	fmt.Fprint(textout, par.Prefix)
-	fmt.Fprintf(textout, "creating new chain. Owner address: %s. State controller: %s, N = %d, T = %d\n",
+	fmt.Fprintf(textout, "Creating new chain\n* Owner address:    %s\n* State controller: %s, N = %d, T = %d\n",
 		originatorAddr, stateControllerAddr, par.N, par.T)
```
Comment (dessaya) [inline packages/apilib/deploychain.go:46]: Why not call `Fprintf` once for each line? For readability.

Also, maybe change `N = %d, T = %d` to something more friendly? Like `Committee size:` and `Quorum:`

**Review by dessaya — APPROVED** [review]

File: packages/apilib/deploychain.go (hunk 1 above)
Comment (howjmay) [reply packages/apilib/deploychain.go:46]: Who can I ask about the meaning of N and T? I am not sure about the meaning. 
I will change the `Fprintf` part

Comment (kape1395) [reply packages/apilib/deploychain.go:46]: `N` is the committee's size, and `T` is the committee's threshold -- a minimal number of committee nodes is needed for the chain to proceed.

We should use consistent terms in the CLI, at least. Either quorum or threshold in all places. Maybe quorum is used now more widely.

Comment (howjmay) [reply packages/apilib/deploychain.go:46]: I am kind not sure how to change it.
Should I change `N` to `size`, and `T` to `threshold`?

Comment (dessaya) [reply packages/apilib/deploychain.go:46]: IMO "Committee size" and "Quorum" are better.

**Pull request merged**